### PR TITLE
Rename Roles In Code

### PR DESCRIPTION
**Pull request by Tommy-XL**

### CODE REVIEW

**Review by 0xDrMoe — APPROVED** [review]
All looks good for these

**Review by Tommy-XL — COMMENTED** [review]
Everything seems fine

File: Modules/CustomRolesHelper.cs [hunk 1]
```diff
@@ -1508,12 +1508,12 @@ public static bool CheckAddonConfilct(CustomRoles role, PlayerControl pc, bool c
...
                     || pc.Is(CustomRoles.Mafia)
                     || pc.Is(CustomRoles.Sniper)
-                    || pc.Is(CustomRoles.FireWorks)
+                    || pc.Is(CustomRoles.Fireworker)
                     || pc.Is(CustomRoles.Ludopath)
                     || pc.Is(CustomRoles.Swooper)
                     || pc.Is(CustomRoles.Vampire)
                     || pc.Is(CustomRoles.Vampiress)
-                    || pc.Is(CustomRoles.Sans)
+                    || pc.Is(CustomRoles.Arrogance)
```
Comment (Ultradragon005) [inline Modules/CustomRolesHelper.cs:1516]: https://i.pinimg.com/originals/48/8d/24/488d24c1b1ac3233b6c56f4c45a53d7d.gif

Comment (Ultradragon005) [reply Modules/CustomRolesHelper.cs:1516]: image

**Review by 0xDrMoe — APPROVED** [review]

File: main.cs [hunk 2]
```diff
@@ -40,14 +40,14 @@ public class Main : BasePlugin
     public static readonly string MainMenuText = " ";
 
     public const string PluginGuid = "com.0xdrmoe.townofhostenhanced";
-    public const string PluginVersion = "2024.0128.151.04"; // YEAR.MMDD.VERSION.CANARYDEV
-    public const string PluginDisplayVersion = "1.5.1 Dev 4";
+    public const string PluginVersion = "2024.0128.151.2"; // YEAR.MMDD.VERSION.CANARYDEV
+    public const string PluginDisplayVersion = "1.5.1 Canary 2";
```
Comment (Tommy-XL) [inline main.cs:44]: Should be 1.5.0 Canary 2 i guess

**Comment by Tommy-XL** [comment]
Oh well, lol

**Pull request merged**